scripts/frequency-start.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+if [ -z "${SEALING_MODE}" ]
+then
+    SEALING_MODE=instant
+fi
+
+if [ -n "${SEALING_INTERVAL}" ]
+then
+    SEALING_INTERVAL="--sealing-interval=${SEALING_INTERVAL}"
+fi
+
+if [ "${CREATE_EMPTY_BLOCKS}" = true ]
+then
+    CREATE_EMPTY_BLOCKS="--sealing-create-empty-blocks"
+fi
+
+exec /frequency/frequency \
```

Comment:
Noting the path - is this script intended -only- for dockerfiles then?

Reply:
Yes, only for dockerfiles. You thinking maybe to put it in the `docker` directory?

Reply:
I think moving it to the docker directory is a good idea.

Reply:
Done.